src/components/nabla/Pools/Backstop/index.tsx

```diff
@@ -12,7 +12,7 @@ const BackstopPoolsBody = (): JSX.Element | null => {
   const { data, isLoading } = useBackstopPools();
 
   if (isLoading) return <Skeleton className="bg-neutral-200 h-48 w-full" />;
-  const pool = data?.[0];
+  const pool = data?.[data.length - 1];
```

Comment:
Why do we use the last one? Because we assume that it's the latest deployed backstop pool?

Reply:
Yeah. This is not correct either way. We need a proper way of filtering the pools but I do believe the most recent comes in last so for testing this will be more correct.